### PR TITLE
Add update site

### PR DESCRIPTION
**Pull request by tatianacv**
Adding files necessary to create Pydev's updatesite.

### CODE REVIEW

File: org.python.pydev.updatesite/site.xml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<site>
+   <description name="Ponder Custom PyDev Update Site" url="https://raw.githubusercontent.com/tatianacv/Pydev/tree/pydev_9_3_1/org.python.pydev.updatesite">
```
Comment (khatchad) [inline org.python.pydev.updatesite/site.xml]: Change to `https://raw.githubusercontent.com/tatianacv/Pydev/pydev_9_3_1/org.python.pydev.updatesite`

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-name: Tests - PyDev
+name: Build with Maven
```
Comment (khatchad) [inline .github/workflows/build.yml]: Pls revert this file.

File: .project [hunk 3]
```diff
@@ -5,6 +5,11 @@
 	<projects>
 	</projects>
 	<buildSpec>
+		<buildCommand>
```
Comment (khatchad) [inline .project]: Pls revert this file because it doesn't seem to have anything to do with the purpose of this PR.

File: .pydevproject [hunk 4]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (khatchad) [inline .pydevproject]: Same here.

File: org.python.pydev.updatesite/.project [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+			<name>org.eclipse.m2e.core.maven2Builder</name>
+			<arguments>
+			</arguments>
+		</buildCommand>
+	</buildSpec>
+	<natures>
+		<nature>org.eclipse.m2e.core.maven2Nature</nature>
+		<nature>org.eclipse.pde.UpdateSiteNature</nature>
+	</natures>
+</projectDescription>
```
Comment (khatchad) [inline org.python.pydev.updatesite/.project]: Need an empty last line here.

File: org.python.pydev.updatesite/site.xml [hunk 6]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<site>
+   <description name="Ponder Custom PyDev Update Site" url="https://raw.githubusercontent.com/tatianacv/Pydev/pydev_9_3_1/org.python.pydev.updatesite">
```
Comment (khatchad) [inline org.python.pydev.updatesite/site.xml]: Change to `name=PyDev Update Site`.

File: plugins/org.python.pydev.updatesite/.project [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+		<buildCommand>
+			<name>org.eclipse.pde.UpdateSiteBuilder</name>
+			<arguments>
+			</arguments>
+		</buildCommand>
+	</buildSpec>
+	<natures>
+		<nature>org.eclipse.pde.UpdateSiteNature</nature>
+	</natures>
+</projectDescription>
```
Comment (khatchad) [inline plugins/org.python.pydev.updatesite/.project]: Newline.

File: plugins/org.python.pydev.updatesite/pom.xml [hunk 8]
```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (khatchad) [inline plugins/org.python.pydev.updatesite/pom.xml]: Remove this.

**Review by khatchad — COMMENTED** [review]
Don't rename files. We have to remove the JARs that are in this PR right now, because we need to first bump the version number. So, remove those in this PR. In a subsequent PR, we'll do the release. This PR is just for the update site.

**Comment by tatianacv** [comment]
It is currently building, the errors is on the Testing with maven.

**Comment by khatchad** [comment]
Thanks!

**Pull request merged**